drivers/pwm/Kconfig.max32

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2023-2024 Analog Devices, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+config PWM_MAX32
+	bool "MAX32 PWM driver"
+	default y
+	depends on DT_HAS_ADI_MAX32_PWM_ENABLED
```

Comment:
PINCTRL should be selected too.
select PINCTRL

Reply:
Done